### PR TITLE
Adopt new CSS library version and migrate shell, masthead and navigation to CSS custom properties

### PR DESCRIPTION
**Pull request by PatoBeltran**
This is the first step in the migration to the new colors in the fluent control library.

### CODE REVIEW

File: lib/components/Masthead/Masthead.module.scss [hunk 1]
```diff
@@ -72,23 +38,43 @@ $search-max-size: 100 * $grid-size;
     .nav-container {
         // show nav in masthead only in small screens:
         display: none;
+        color: var(--color-text-masthead);
 
-        @media(max-width: $screen-sm - 1) {
-            width: $layout-nav-collapsed-width;
-            display: initial;
+        .nav-panel {
+            width: $layout-nav-expanded-width;
```
Comment (vishwam) [inline lib/components/Masthead/Masthead.module.scss]: var

File: lib/components/Masthead/Masthead.module.scss [hunk 2]
```diff
@@ -72,23 +38,43 @@ $search-max-size: 100 * $grid-size;
...
+            color: var(--color-text-regular);
+
+            &:hover,
+            &:active {
+                background-color: var(--color-bg-masthead-hover);
+            }
+        }
+
+        @media(max-width: $screen-sm - 1) {
+            width: $layout-nav-collapsed-width;
```
Comment (vishwam) [inline lib/components/Masthead/Masthead.module.scss]: var

File: lib/components/Masthead/Masthead.module.scss [hunk 3]
```diff
@@ -169,7 +114,6 @@ $search-max-size: 100 * $grid-size;
             justify-content: flex-end;
             list-style-type: none;
 
-
             // hide the search-button that control the display of search-bar
             .search-button {
                 @media(min-width: $screen-md) {
```
Comment (vishwam) [inline lib/components/Masthead/Masthead.module.scss]: var

Comment (PatoBeltran) [reply lib/components/Masthead/Masthead.module.scss]: `var(...)` can only be used as properties, therefore media query conditions cannot be set with `var(...)`... for this we have to keep using scss vars

File: lib/components/Masthead/Masthead.module.scss (hunk 1 above)
Comment (PatoBeltran) [reply lib/components/Masthead/Masthead.module.scss]: synced offline. Constants are kept as scss variables.

**Pull request merged**